cli/command_test.go

```diff
@@ -314,7 +314,7 @@ func (c *TestCommand) Run(ctx context.Context, args []string) error {
 	}
 
 	if v := c.Output; v != "" {
-		fmt.Fprint(c.Stdout(), v)
+		c.Outf(v)
```

Comment:
I think there might be a few examples to clean up too?

Reply:
Done